src/test/feature/Ranger/test_ranger.cpp

```diff
@@ -24,19 +24,34 @@
 
 #include "lib/command.h"
 #include "lib/gpfdist.h"
-#include "lib/sql_util.h"
 #include "lib/string_util.h"
 
 using std::vector;
 using std::string;
 using hawq::test::SQLUtility;
 using hawq::test::Command;
 
+TestHawqRanger::TestHawqRanger()
+{
+	initfile = hawq::test::stringFormat("Ranger/sql/init_file");
+	rangerHost = getRangerHost();
+}
+
+std::string& TestHawqRanger::getRangerHost()
```

Comment:
And there is function "getGUCValue" to get the "hawq_rps_address_host".
Besides, "hawq_rps_address_host" is the rps address host, not ranger host, so what do you wanna to get ? If ranger server host, then they are not the same.